ec/ecresource/deploymentresource/schema.go

```diff
@@ -22,9 +22,10 @@ import (
 )
 
 const (
-	minimumKibanaSize           = 1024
-	minimumApmSize              = 512
-	minimumEnterpriseSearchSize = 2048
+	minimumKibanaSize             = 1024
+	minimumApmSize                = 512
+	minimumEnterpriseSearchSize   = 2048
+	minimumIntegrationsServerSize = 1024
```

Comment:
I've changed the min size for the integrations server since APM will oom and crash with 512.

Reply:
👍 (size at the platform's end starts with 1024 as well)